INFO     HTTP_LOGGER:event_hooks.py:31 Выполняем POST запрос к http://localhost:8000/api/v1/users
INFO     HTTP_LOGGER:event_hooks.py:40 Получен ответ 200 OK от http://localhost:8000/api/v1/users
INFO     HTTP_LOGGER:event_hooks.py:31 Выполняем POST запрос к http://localhost:8000/api/v1/authentication/login
INFO     HTTP_LOGGER:event_hooks.py:40 Получен ответ 200 OK от http://localhost:8000/api/v1/authentication/login
INFO     HTTP_LOGGER:event_hooks.py:31 Выполняем GET запрос к http://localhost:8000/api/v1/files/incorrect-file-id
INFO     HTTP_LOGGER:event_hooks.py:40 Получен ответ 422 Unprocessable Entity от http://localhost:8000/api/v1/files/incorrect-file-id
INFO     BASE_ASSERTIONS:base.py:20 Проверяем что статус-код ответа соответствует 422
INFO     FILES_ASSERTIONS:files.py:134 Проверка ответа на получение файла с некорректным идентификатором файла
INFO     ERRORS_ASSERTIONS:errors.py:39 Проверка ошибки из ответа
INFO     BASE_ASSERTIONS:base.py:73 Проверяем что длина 'details' соответствует 1
INFO     ERRORS_ASSERTIONS:errors.py:21 Проверка ошибки валидации
INFO     BASE_ASSERTIONS:base.py:38 Проверяем что 'type' соответствует uuid_parsing
INFO     BASE_ASSERTIONS:base.py:38 Проверяем что 'input' соответствует incorrect-file-id
INFO     BASE_ASSERTIONS:base.py:38 Проверяем что 'context' соответствует {'error': 'invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `i` at 1'}
INFO     BASE_ASSERTIONS:base.py:38 Проверяем что 'message' соответствует Input should be a valid UUID, invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `i` at 1
INFO     BASE_ASSERTIONS:base.py:38 Проверяем что 'location' соответствует ['path', 'file_id']
INFO     SCHEMA_ASSERTIONS:schema.py:25 Валидация JSON схемы